docs/troubleshooting.md

```diff
@@ -155,6 +155,8 @@ how to run this setup from a webserver running on App Engine Flex (Node).
 
 Running Puppeteer on Heroku requires some additional dependencies that aren't included on the Linux box that Heroku spins up for you. To add the dependencies on deploy, add the Puppeteer Heroku buildpack to the list of buildpacks for your app under Settings > Buildpacks.
 
+Simple Guide: https://timleland.com/headless-chrome-on-heroku/
```

Comment:
I don't mind additional helpful resources, but can you move this to the bottom of the section? It's in the middle of the step by step explainer.

Maybe:
`There's also another [simple guide](https://timleland.com/headless-chrome-on-heroku/) from @timleland includes a sample project: https://timleland.com/headless-chrome-on-heroku/.`

Reply:
Good idea. Made the changes. Thanks